### DISCUSSION TITLE
Watchtower updates Jackett, wipes API key/config

### DISCUSSION THREAD

**Discussion by peanutlasko** [containrrr/watchtower#883] [Q&A] [answered]
Hi,

I have a homelab with Watchtower configured - it runs perfectly fun, updates every 12 hours without issue.

I can destroy my jackett image and container, re-run with Docker-compose and my API key will remain the same.

However, when a new image is detected and Watchtower automatically finds this, stops the old container, updates, and creates the new jackett container - a new API key is created, which breaks all my integrations.

My /config is not mounted to an Named Volume. I have a bind mount created on my host computer that I can see and confirm it does not get destroyed with the container is deleted. this only happens when a new jackett image is found.

**Comment by simonaronsson** [comment]
I've been able to use jackett with watchtower without any particular issues. I would recommend that you make sure the API token is indeed stored in that config file. Without seeing your docker-compose file it's hard to know what might be the issue. Would you mind posting a redacted version in this thread?

**Reply by peanutlasko** [reply]
sure, attached:

[docker-compose copy.txt](https://github.com/containrrr/watchtower/files/6265797/docker-compose.copy.txt)

One thing to note - I have Jackett setup right now as "monitor only" because of this issue. I'm handling the updates myself manually.

**Reply by peanutlasko** [reply]
Ok now I'm really losing my mind. Jackett API is changing randomly. Not because of watchtower. I specifically have not updated it, the container has not even stopped running, but today the API key is different. Hmm something weird is going on.

I wonder if I need to assign Jackett a static IP address ?

**Comment by peanutlasko** [comment, marked as answer]
Figured out this was a permission issue - Jackett kept remaking the config
